home.admin/config.scripts/lnd.install.sh

```diff
@@ -306,13 +306,20 @@ debuglevel=info
 gc-canceled-invoices-on-startup=true
 gc-canceled-invoices-on-the-fly=true
 ignore-historical-gossip-filters=1
-sync-freelist=true
 stagger-initial-reconnect=true
 tlsautorefresh=1
 tlsdisableautofill=1
 tlscertpath=/home/bitcoin/.lnd/tls.cert
 tlskeypath=/home/bitcoin/.lnd/tls.key
 
+# Set to false for nodes with larger amount of channels. This modification leads to increased 
+# latency during initialization, yet significantly boosts runtime performance of the daemon.
+sync-freelist=true
+
+# Specify the maximum number of logfiles retained in rotation and the threshold size for rotation initiation.
+maxlogfiles=7
+maxlogfilesize=500
```

Comment:
this would result to 7 * 500 MB = 3.5 GB LND logfiles. Is that the intended size?
Would be happy with rotating them for every 100 MB at least and keep maybe 5 so 500 MB max, what so you think?
```suggestion
maxlogfiles=5
maxlogfilesize=100
```

Reply:
Note that the 500MB is before the logrotate
`# Max log file size in MB before it is rotated.` 

My logfiles after gz span between 25-80MB, so we talk 7x80MB = 560MB max
How about 5*400?

```
sudo ls -lah ~/.lnd/logs/bitcoin/mainnet
total 856M
drwxrwxr-x 2 lnd lnd 4.0K Feb 25 13:10 .
drwx-w---- 3 lnd lnd 4.0K Mar  2  2022 ..
-rw-r--r-- 1 lnd lnd  66M Feb 26 21:48 lnd.log
-rw-r--r-- 1 lnd lnd  25M Jan 22 17:12 lnd.log.2707.gz
-rw-r--r-- 1 lnd lnd  23M Jan 22 19:00 lnd.log.2708.gz
-rw-r--r-- 1 lnd lnd  26M Jan 22 20:54 lnd.log.2709.gz
-rw-r--r-- 1 lnd lnd  26M Jan 22 22:55 lnd.log.2710.gz
-rw-r--r-- 1 lnd lnd  26M Jan 23 00:53 lnd.log.2711.gz
-rw-r--r-- 1 lnd lnd  26M Jan 23 02:28 lnd.log.2712.gz
-rw-r--r-- 1 lnd lnd  26M Jan 23 04:33 lnd.log.2713.gz
-rw-r--r-- 1 lnd lnd  26M Jan 23 06:49 lnd.log.2714.gz
-rw-r--r-- 1 lnd lnd  28M Jan 23 09:21 lnd.log.2715.gz
-rw-r--r-- 1 lnd lnd  26M Jan 23 11:40 lnd.log.2716.gz
-rw-r--r-- 1 lnd lnd  27M Jan 23 13:49 lnd.log.2717.gz
-rw-r--r-- 1 lnd lnd  30M Jan 23 15:59 lnd.log.2718.gz
-rw-r--r-- 1 lnd lnd  26M Jan 23 17:44 lnd.log.2719.gz
-rw-r--r-- 1 lnd lnd  27M Jan 23 19:38 lnd.log.2720.gz
-rw-r--r-- 1 lnd lnd  24M Jan 23 21:26 lnd.log.2721.gz
-rw-r--r-- 1 lnd lnd  23M Jan 23 23:30 lnd.log.2722.gz
-rw-r--r-- 1 lnd lnd  22M Jan 24 01:28 lnd.log.2723.gz
-rw-r--r-- 1 lnd lnd  24M Jan 24 03:38 lnd.log.2724.gz
-rw-r--r-- 1 lnd lnd  26M Jan 24 05:39 lnd.log.2725.gz
-rw-r--r-- 1 lnd lnd  28M Jan 24 07:52 lnd.log.2726.gz
-rw-r--r-- 1 lnd lnd  65M Jan 31 10:41 lnd.log.2727.gz
-rw-r--r-- 1 lnd lnd  69M Feb  7 07:22 lnd.log.2728.gz
-rw-r--r-- 1 lnd lnd  73M Feb 15 04:51 lnd.log.2729.gz
-rw-r--r-- 1 lnd lnd  80M Feb 25 13:10 lnd.log.2730.gz
```